tools/star.wdl

```diff
@@ -585,20 +585,20 @@ task alignment {
         Int alignWindowsPerReadNmax = 10000
         Int alignTranscriptsPerWindowNmax = 100
         Int alignTranscriptsPerReadNmax = 10000
-        Int peOverlapNbasesMin = 0
+        Int peOverlapNbasesMin = 10
```

Comment:
This purports to improve accuracy for paired end calling by turning on the "§ 10. Merging and mapping of overlapping paired-end reads." Reading over the algorithm, it seems fine to include in RNA-Seq v2 workflows, though I still might stick with the defaults.